packages/dev/serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -811,11 +811,7 @@ export class GLTFExporter {
 
         // Scene metadata
         if (this._babylonScene.metadata) {
-            if (this._options.metadataSelector) {
-                scene.extras = this._options.metadataSelector(this._babylonScene.metadata);
-            } else if (this._babylonScene.metadata.gltf) {
-                scene.extras = this._babylonScene.metadata.gltf.extras;
-            }
+            scene.extras = this._options.metadataSelector(this._babylonScene.metadata);

```

Comment:
nit: Just FYI. This will set `scene.extras` to `undefined` which is technically different than not having `scene.extras`. The JSON for the glTF won't be different since JSON doesn't allow `undefined` so it's probably fine. If you want to be extra safe, change to the following. Ditto for the other one.
```suggestion
            const extras = this._options.metadataSelector(this._babylonScene.metadata);
            if (extras) {
                scene.extras = extras;
            }
```